recipes/yaqc-bluesky/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "yaqc-bluesky" %}
+{% set version = "2020.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 443c9cab3d4b7675e2e8aa4105e81241e8846ad34cfa329703c3f4c4e61d2a9f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit
+    - yaqc >=2020.7.3
```

Comment:
I'm not 100% clear if `yaqc` is needed in the `host` section. Does `flit` need what is listed in `requires` in `tool.flit.metadata` to actually build the package, or is it just a run time requirement?

Reply:
`flit` needs everything at build time---strange but true